ios/RNIapIos.m

```diff
@@ -315,6 +315,7 @@ -(NSDictionary*)getProductObject:(SKProduct *)product {
 	
   NSString* localizedPrice = [formatter stringFromNumber:product.price];
   NSString* introductoryPrice = localizedPrice;
+  NSString* locale = localeIdentifierToBCP47(product.priceLocale.localeIdentifier);
```

Comment:
`RNIapIos.m:318:22`: Implicit conversion of 'int' to 'NSString *' is disallowed with ARC

Here is another `conversion` problem that have trouble with building.